tests/framework/db/CommandTest.php

```diff
@@ -298,6 +298,15 @@ public function testBatchInsert()
         $this->assertEquals(0, $command->execute());
     }
 
+    public function testBatchInsertWithYield()
+    {
+        if (version_compare(PHP_VERSION, '5.5', '<')) {
+            $this->markTestSkipped('The yield function is only supported with php 5.5 =< version');
+        } else {
+            include __DIR__ . '/testBatchInsertWithYield.php';
```

Comment:
Yes, of course)
`yield` is not valid syntax for php < 5.5
otherwise, it is impossible to run a test for php < 5.5